routes/movieRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const movieController = require('../controllers/movieController');
+const router = express.Router();
+
+router.get('/main', movieController.getMain);
+router.get('/detail/:movieId', movieController.getDetail);
```

Comment:
```suggestion
router.get('/:movieId', movieController.getDetail);
```
`:3000/movies/1`도 충분합니다.

Reply:
네 반영하겠습니다